node/pkg/processor/cleanup.go

```diff
@@ -81,16 +80,17 @@ func (p *Processor) handleCleanup(ctx context.Context) {
 					p.logger.Info("Expiring late VAA", zap.String("digest", hash), zap.Duration("delta", delta))
 					aggregationStateLate.Inc()
 					delete(p.state.signatures, hash)
-					break
+					continue
```

Comment:
looking at this code, `continue` _seems_ to be the right thing to do. Do you know why it did a `break` before?

Reply:
It was an early break out of the switch case.

Reply:
makes sense, the first resource I found on `break` only discussed loops, but but now I see that it applies to switch and select too